runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldHopperPlugin.java

```diff
@@ -504,6 +509,7 @@ private void updateList()
 		WorldResult worldResult = worldService.getWorlds();
 		if (worldResult != null)
 		{
+			setWestCoastWorlds();
```

Comment:
this makes it so updateList() must be called on the client thread, but it called off-thread from the config change event handler still.